### PR TITLE
TST: Fix instruct-blip tests

### PR DESCRIPTION
**Pull request by younesbelkada**
# What does this PR do?

With the recent CI update to torch 2.3, this PR fixes the expected results for instruct-blip tests

cc @ydshieh

### CODE REVIEW

**Review by ArthurZucker — COMMENTED** [review]
Is this related potentially to fp32 flags that we don't set? Or just breaking changes from torch 2.3?

**Comment by younesbelkada** [comment]
I'll let @ydshieh comment on this but I don't think it's a breaking change, it often happens that some logits / generated texts slightly differ from one minor version to another

**Comment by ydshieh** [comment]
`InstructBlipModelIntegrationTest::test_inference_vicuna_7b` is from the torch version.

`tests/models/instructblip/test_modeling_instructblip.py::InstructBlipModelIntegrationTest::test_inference_flant5_xl` is not, it fails since Dec. 9 2023, but I can't really see what's going on there.

In any case, it's not the fp32 that is fixed in #31018 27867

**Review by ydshieh — APPROVED** [review]
Thanks for updating!

**Comment by younesbelkada** [comment]
OK thanks ! Merging !

**Comment by ydshieh** [comment]
Yeah, finally got it, it is caused by #27808 (commit `b31905d1`)

(`test_inference_flant5_xl `)
